scripts/components/CollectionList.js

```diff
@@ -123,13 +145,16 @@ class Table extends Component {
         </thead>
         <tbody>{
           records.map((record, index) => {
-            return <Row key={index}
-              name={name}
-              record={record}
-              schema={schema}
-              config={config}
-              deleteRecord={deleteRecord}
-              updatePath={updatePath} />;
+            return (
+              <Row key={index}
+                name={name}
+                record={record}
+                conflict={conflicts[record.id]}
```

Comment:
Wait, how are conflicts returned? I assumed it was just a list, but it looks more like a Map or something here.